infra/ansible/playbooks/telemetry.yaml

```diff
@@ -0,0 +1,56 @@
+- name: Run Setup playbook
+  ansible.builtin.import_playbook: setup.yaml
+  vars:
+    host: telemetry
+
+- name: Run Elixir playbook
+  ansible.builtin.import_playbook: elixir.yaml
+  vars:
+    host: telemetry
+
+- name: Run Caddy playbook
+  ansible.builtin.import_playbook: caddy.yaml
+  vars:
+    host: telemetry
+    ini_file: ini/config-telemetry.ini # add template for caddyfile for grafana (public) and Jaeger (only tailscale)
+
+- name: Run Prometheus playbook
+  ansible.builtin.import_playbook: prometheus.yaml
+  vars:
+    host: telemetry
+    prometheus_version: "2.53.2"
+    ini_file: ini/config-telemetry.ini
+
+- name: Run Grafana playbook
+  ansible.builtin.import_playbook: grafana.yaml
+  vars:
+    host: telemetry
+    ini_file: ini/config-telemetry.ini
+    TODO: add dashboards
+
+- name: Run OpenTelemetry playbook
+  ansible.builtin.import_playbook: open_telemetry.yaml
+  vars:
+    host: telemetry
+
+- name: Run Jaeger playbook
+  ansible.builtin.import_playbook: jaeger.yaml
+  vars:
+    host: telemetry
+    version: 1.63.0
+
+- name: Run Cassandra playbook
+  ansible.builtin.import_playbook: cassandra.yaml
+  vars:
+    host: telemetry
+
+- name: Run Postgres playbook
+  ansible.builtin.import_playbook: postgres.yaml
+  vars:
+    host: telemetry
```

Comment:
@JuArce We need to discuss whether we would like to have each component on it's independent file, or everything on a single one. 

I did it this way since everything else was also segmented, but we might want to chunk all of this onto the same file (this one, `telemetry.yaml)